docs/pages/desktop-access/active-directory-manual.mdx

```diff
@@ -682,6 +682,49 @@ Computer Configuration > Policies > Windows Settings > Security Settings > Local
 Double click the "Add workstations to domain" policy and ensure that the
 "Authenticated Users" group is not present.
 
+## Multiple Domains
+
+Each `windows_desktop_service` is designed to support connecting to hosts in a
+single Active Directory domain. If you have multiple independent domains, you
+can deploy multiple Teleport agents to service them.
+
+If you have multiple domains with a trust relationship between them, you can
+configure Teleport to perform PKI operations against a different domain.
+
+In order for this to work, the hosts that you want to connect to and the AD
+users that you want to connect as must reside in the same domain.
+
+For example, suppose you have a root domain at `example.com` and a child domain
+for developers at `dev.example.com`. If your PKI is configured at the root, but
+you want to allow users in the child domain to connect to hosts in the child
+domain, you would do the following:
+
+First, import Teleport's CA certificate as a trusted root certificate in the root
+domain's group policy and add the certificate to the NTAuth store as listed above.
+
+Next, configure Teleport to perform PKI against the root domain, while issuing
+certificates for users and hosts in the child domain:
+
+```yaml
+windows_desktop_service:
+  enabled: yes
+
+  # configure LDAP settings to point at the child domain
+  ldap:
+    addr: dev.example.com:636
+    username: 'DEV\svc-teleport'
+
+  # optional: configure discovery for the child domain
+  discovery:
+    base_dn: CN=Computers,DC=dev,DC=example,DC=com
+
+  # perform PKI against the root domain
+  pki_domain: root.example.com
+```
+
+With this configuration, Teleport will generate certificates for users in
+`dev.example.com`, but it will publish its CA and CRLs to `example.com`.
+
```

Comment:
Perhaps an `<Admonition type="warning" title="something">` warning that this won't work for other "types" of trust relationships, e.g. I'm inferring that it won't work if you have users at `another-example.com` because it's not a subdomain (though maybe that's incorrect).

Reply:
A subdomain is a DNS concept. I don't think it particularly matters what the name of the domain is so long as the trust is set up.